### PR TITLE
Ana/add no proposal history

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Integrating the Regen network into the frontend I noticed that there was an issue with the `PageProposals` component: the banner of "No Governance History" wouldn't be displayed, and the component would just stay blank, due to lack of proposals history.

Luckily I fixed it by removing the `PageContainer` altogether.

This is the screen as it was before the fix:

image

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [x] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/governance/PageProposals.vue [hunk 1]
```diff
@@ -1,56 +1,48 @@
...
+        </div>
+        <div slot="subtitle">
+          There are currently no governance proposals to display.
+          Click the 'Create Proposal' button to submit the first proposal of the network!
+        </div>
+      </TmDataMsg>
+    </div>
+  </div>
+  <div v-else-if="!$apollo.loading && proposals.length > 0">
+    <div class="button-container">
```
Comment (faboweb) [inline src/components/governance/PageProposals.vue]: The button is duplicated, can we pull it out?

File: src/components/governance/PageProposals.vue [hunk 2]
```diff
@@ -1,56 +1,48 @@
...
-        @success="() => afterPropose()"
+    </div>
+    <div>
+      <TmDataMsg icon="gavel">
+        <div slot="title">
+          No Governance Proposals
+        </div>
+        <div slot="subtitle">
+          There are currently no governance proposals to display.
+          Click the 'Create Proposal' button to submit the first proposal of the network!
```
Comment (jbibla) [inline src/components/governance/PageProposals.vue]:
```suggestion
          Click the 'Create Proposal' button to submit the first network proposal!
```

File: src/components/governance/PageProposals.vue [hunk 3]
```diff
@@ -1,56 +1,48 @@
 <template>
-  <PageContainer
```
Comment (jbibla) [inline src/components/governance/PageProposals.vue]: what was `PageContainer` doing?

Comment (colw) [reply src/components/governance/PageProposals.vue]: > what was `PageContainer` doing?

It was intended to replace TmPage and some of the internals we don't need there anymore.

I assume we will still need TmPage to be present?

Comment (faboweb) [reply src/components/governance/PageProposals.vue]: I think so true. I think it should be like:
```
<TmPage
  data-title="Proposals"
  :managed="false">
...
```

Comment (Bitcoinera) [reply src/components/governance/PageProposals.vue]: Ok, I am going to change it

File: src/components/governance/PageProposals.vue (hunk 2 above)
Comment (Bitcoinera) [reply src/components/governance/PageProposals.vue]: The language of Shakespeare and its mysteries...

File: src/components/governance/PageProposals.vue (hunk 1 above)
Comment (Bitcoinera) [reply src/components/governance/PageProposals.vue]: Done

**Comment by Bitcoinera** [comment]
Ok, now it should be good

**Comment by Bitcoinera** [comment]
Oops, I missed the unit tests

File: src/components/governance/PageProposals.vue [hunk 4]
```diff
@@ -1,56 +1,45 @@
...
+          </div>
+          <div slot="subtitle">
+            There are currently no governance proposals to display.
+            Click the 'Create Proposal' button to submit the first network proposal!
+          </div>
+        </TmDataMsg>
       </div>
+    </div>
+    <div v-else-if="!$apollo.loading && proposals.length > 0">
       <TmDataLoading v-if="$apollo.loading" />
```
Comment (faboweb) [inline src/components/governance/PageProposals.vue]: this contradicts line 31

File: src/components/governance/PageProposals.vue [hunk 5]
```diff
@@ -1,56 +1,45 @@
...
+            No Governance Proposals
+          </div>
+          <div slot="subtitle">
+            There are currently no governance proposals to display.
+            Click the 'Create Proposal' button to submit the first network proposal!
+          </div>
+        </TmDataMsg>
       </div>
+    </div>
+    <div v-else-if="!$apollo.loading && proposals.length > 0">
```
Comment (faboweb) [inline src/components/governance/PageProposals.vue]:
```suggestion
    <div v-else>
```

File: src/components/governance/PageProposals.vue (hunk 4 above)
Comment (faboweb) [reply src/components/governance/PageProposals.vue]: you can move it up and use a `v-else-if`

**Pull request merged**